inc/admin/upgrader.php

```diff
@@ -297,7 +284,6 @@ function rocket_new_upgrade( $wp_rocket_version, $actual_version ) {
 	if ( version_compare( $actual_version, '2.9.5', '<' ) ) {
 		if ( is_plugin_active( 'autoptimize/autoptimize.php' ) ) {
 			if ( 'on' === get_option( 'autoptimize_html' ) ) {
-				update_rocket_option( 'minify_html', 0 );
 				update_rocket_option( 'minify_html_inline_css', 0 );
 				update_rocket_option( 'minify_html_inline_js', 0 );
```

Comment:
You can remove those too, as it's linked to the `minify_html` option, and in fact was removed some time ago

Reply:
Thank you! I missed this one